devops/PyPI-Release.yml

```diff
@@ -1,200 +0,0 @@
-# Simplified PyPI release pipeline
```

Comment:
To confirm, we do not use this in any other releases?

Reply:
yes, just for interpret-community